### ISSUE TITLE
Doc is really not up to date ?

### ISSUE THREAD

**Issue by manodupont** [jquense/react-bootstrap-modal#39]
So i looked into the code of the latest release,

there is no onEnter, onEntered, there is only onEntering.

There might be some other flaws... am i correct ?

**Comment by jquense** [comment]
everything looks up to date to me, both in the repo and on `npm`. I see all the exit and enter events in both places

**Comment by manodupont** [comment]
My point is, the doc says that, but the code is not there.

There is no onEnter and onEntered and onExit etc method in the code.


Sent from my iPhone

On Mar 10, 2017, at 13:17, Jason Quense <notifications@github.com<mailto:notifications@github.com>> wrote:


everything looks up to date to me, both in the repo and on npm. I see all the exit and enter events in both places

**Comment by jquense** [comment]
yes there is? https://github.com/jquense/react-bootstrap-modal/blob/master/src/Modal.js#L197-L202

**Comment by manodupont** [comment]
Mmm i will check again but i ended up looking into the code to find them cause they weren't called. I only got onHide to call an handler.



Sent from my iPhone

On Mar 10, 2017, at 14:21, Jason Quense <notifications@github.com<mailto:notifications@github.com>> wrote:


yes there is? https://github.com/jquense/react-bootstrap-modal/blob/master/src/Modal.js#L197-L202

**Comment by jquense** [comment]
they are transition hooks, so they only fire if you have an animation
